openstack-py-webserver/README.md

```diff
@@ -0,0 +1,71 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Web Server Using Openstack
+
+
+## Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+1. [Configure Pulumi for Openstack](https://www.pulumi.com/docs/intro/cloud-providers/openstack/setup/)
+1. [Configure Pulumi for Python](https://www.pulumi.com/docs/intro/languages/python/)
+
+## Deploying and running the program
+
+1. Create a new stack:
+
+    ```bash
+    $ pulumi stack init
+    ```
+
+2. Modify `__main__.py` to include your keypair and image
+
+3. Run `pulumi up` to preview and deploy changes:
+
+    ```bash
+    $ pulumi up
+    Previewing update (dev):
+        Type                                  Name           Plan
+        pulumi:pulumi:Stack                   pulum-dev
+    +   ├─ openstack:images:Image             fedora         create
+    +   ├─ openstack:compute:Keypair          default        create
+    +   ├─ openstack:networking:SecGroupRule  secgroupRule1  create
+    +   ├─ openstack:networking:SecGroupRule  secgroupRule2  create
+    +   ├─ openstack:networking:SecGroupRule  secgroupRule3  create
+    +   └─ openstack:compute:Instance         test_fedora    create
+
+    Outputs:
+    ~ instance_ip: "192.168.0.243" => output<string>
+
+    Resources:
+        + 6 to create
+        1 unchanged
+    Resources:
+        + 6 created
+        1 unchanged
+
+    Duration: 38s
+
+    ```
+
+4. View the host name and IP address of the instance via `stack output`:
+
+    ```bash
+    $ pulumi stack output
+    Current stack outputs (1):
+        OUTPUT       VALUE
+        instance_ip  192.168.0.243
+    ```
+
+5.  Verify that the EC2 instance exists, by either using the AWS Console or running `openstack server list`.
```

Comment:
I think you mean OpenStack instance here?

Reply:
True, Thank you.